cclib/io/moldenwriter.py

```diff
@@ -161,14 +161,15 @@ def _mo_from_ccdata(self):
         if hasattr(self.ccdata, 'mosyms'):
             has_syms = True
             syms = self.ccdata.mosyms
+        else:
```

Comment:
Can you rebase this on top of master? These changes already got merged in from your other PR.

Reply:
Rebased on top of master, no changes to `moldenwriter` in this pull request.